app/lib/submission_builder/document.rb

```diff
@@ -101,6 +101,13 @@ def add_non_zero_value(xml, elem_name, line)
       end
     end
 
+    def add_positive_decimal_value(xml, elem_name, line)
+      value = calculated_fields.fetch(line)
+      if value.present? && value.positive?
+        xml.send(elem_name, value)
+      end
+    end
```

Comment:
i was trying to do this in the submission builder, i guess it's only used it one place so happy to put it back there but maybe it'll be useful for decimal values?

Reply:
i have a question/suggestion that's hopefully not too out of scope: is `add_non_zero_value` really just excluding 0 and intending to fill in negative numbers? if so i think it would make sense to:
A) remove the `to_i` in `add_non_zero_value`, as i don't believe it's serving any purpose
B) rename this to just `add_positive_value`, since it doesn't seem specific to decimals

Reply:
hmm i had to rethink this, I don't think we need a separate method for a positive value, I was just tryign to avoid the `to_i` conversion in `add_non_zero_value` method which is why I created that extra method.
what I actually need is for the `add_non_zero` method to accommodate float values that may come through. I think still want to convert the `value` to float since technically anyone can pass in a string as a value (or a non-numeric value).

Reply:
after brief slack convo with Jenny, here's my decision:
although we could probably use `add_non_zero_float_value` for `value` that is supposed to be an integer type, I separated it out since it's scary to change `add_non_zero` method which is used in so many places, right before launch. Maybe we can revisit though!